src/platform_impl/windows/event_loop.rs

```diff
@@ -25,7 +25,9 @@ use winapi::{
         windowsx, winerror,
     },
     um::{
-        libloaderapi, ole2, processthreadsapi, winbase,
+        libloaderapi,
+        mmsystem::TIMECAPS,
```

Comment:
i suggest using mssystem not mmsystem::TIMECAPS.
seems to be the approach used here.